dev-python/m2crypto/m2crypto-0.31.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6)
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1
+
+MY_PN="M2Crypto"
+
+DESCRIPTION="M2Crypto: A Python crypto and SSL toolkit"
+HOMEPAGE="https://gitlab.com/m2crypto/m2crypto https://pypi.org/project/M2Crypto/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_PN}-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Just ran repoman and noticed, 
```  KEYWORDS.dropped              1
   dev-python/m2crypto/m2crypto-0.31.0.ebuild: alpha amd64-fbsd amd64-linux arm arm64 hppa ia64 mips ppc ppc-macos ppc64 s390 sh sparc x64-macos x86-fbsd x86-linux x86-macos
```

This version bump doesn't seem like a huge one, so you can keep the KEYWORDS from previous ~ version.